website/videoBlog/2024-02-07-Building-an-Open-Source-Data-Lake-House-with-Hudip-Postgres-Hive-Metastore-Minio-and-StarRocks.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "Building an Open Source Data Lake House with Hudi, Postgres Hive Metastore, Minio, and StarRocks"
+last_modified_at: 2024-02-21T16:54:38.964863-07:00
+authors:
+- name: Soumil Shah
+category: blog
+image: /assets/images/video_blogs/2024-02-07-Building-an-Open-Source-Data-Lake-House-with-Hudip-Postgres-Hive-Metastore-Minio-and-StarRocks.png
```

Comment:
The corresponding image file - `2024-02-07-Building-an-Open-Source-Data-Lake-House-with-Hudip-Postgres-Hive-Metastore-Minio-and-StarRocks` is missing from the commit